dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -23,24 +23,25 @@
     padding: 10px;
   }
 
-- if @level.filename
-  = "Filename: #{@level.filename}"
-.field
-  = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :encrypted, description: "Encrypt this entire file"}
-  %p Use this if you want to prevent devious students from finding answers in our git repo
+= render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :encrypted, description: "Encrypt this entire file"}
+%p Use this if you want to prevent devious students from finding answers in our git repo
 
-.script_text
-  = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
+%legend.control-legend{data: {toggle: "collapse", target: "#dsl_editor"}}
+  DSL Editor
 
-- if @level.supports_markdown?
-  - content_for(:head) do
-    %script{src: webpack_asset_path('js/levels/editors/_dsl.js')}
+#dsl_editor.in.collapse
+  .script_text
+    = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
```

Comment:
the editor for evaluation multi contains only these lines: https://github.com/code-dot-org/code-dot-org/blob/f3e52bd6fc2f7cecdbffb14f80de16be4fad7e14/dashboard/app/views/levels/editors/_evaluation_multi.html.haml#L6-L7

as you start to remove more from this dsl partial, it could make sense to just have evaluation multi levels use this dsl partial rather than its own separate partial.

Reply:
Working on in https://github.com/code-dot-org/code-dot-org/pull/35086